config.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"github.com/ddollar/forego/Godeps/_workspace/src/github.com/subosito/gotenv"
```

Comment:
ah wow, not sure what I saw that made me think it hadn't been added yet. Sorry about that :v: